src/components/wallets/WalletAccordion.tsx

```diff
@@ -353,19 +351,3 @@ const DetailSummaryLine = styled.div`
 const TextWithSideSpace = styled(Typography)`
```

Comment:
In summary accordion and content css, can you add the same as [here](https://github.com/getlago/lago-front/pull/266/files#diff-d71335ae21b7b9901ef663a0fa8e944cb50c563af3428bc7c8ee9db07169022fR206) please  ?
(We will update those component once the spec is done)

Reply:
Sure